src/map.cpp

```diff
@@ -4195,7 +4195,7 @@ map_stack::iterator map::i_rem( const tripoint &p, const map_stack::const_iterat
     if( current_submap == nullptr ) {
         debugmsg( "Tried to remove items at (%d,%d) but the submap is not loaded", l.x, l.y );
         nulitems.clear();
-        return map_stack{ &nulitems, p, this }.begin();
+        return map_stack{ &nulitems, p, this } .begin();
```

Comment:
```suggestion
        return map_stack{ &nulitems, p, this }.begin();
```

Reply:
Weirdly enough, this space before `.begin` was put there by astyle.